### DISCUSSION TITLE
Installing Rust Server FAIL

### DISCUSSION THREAD

**Discussion by nikoniko2027** [GameServerManagers/LinuxGSM#3299] [Q&A] [answered]
Hello! Please help me fix the problem!

Screenshot: [https://prnt.sc/10526hx](Screenshot)

**Comment by EugeneGGHQ** [comment, marked as answer]
Hey there,

My first instinct is that it could either be a firewall or a DNS problem (assuming that you're sure you definitely do have an active internet connection on the box!) 

Have you knowingly configured any firewall rules that could block access to SteamCMD download servers? 

To eliminate a DNS issue you might consider updating your server's DNS settings to use something globally recognised (8.8.8.8 for example) and try again. 

Let me know if these suggestions helped.

**Reply by nikoniko2027** [reply]
Hi! Unfortunately, I will not be able to check it, since I changed the server and everything works fine on it! But I'm pretty sure your solution would have helped me :)
